packages/charts/react-charts/library/src/components/CommonComponents/CartesianChart.tsx

```diff
@@ -351,8 +356,9 @@ export const CartesianChart: React.FunctionComponent<ModifiedCartesianChartProps
           yMinValue: props.secondaryYScaleOptions?.yMinValue || 0,
           yMaxValue: props.secondaryYScaleOptions?.yMaxValue ?? 100,
           tickPadding: 10,
-          maxOfYVal: props.secondaryYScaleOptions?.yMaxValue ?? 100,
-          yMinMaxValues: getMinMaxOfYAxis(points, chartType),
+          yMinMaxValues: props.getMinMaxOfYAxis
+            ? props.getMinMaxOfYAxis(points, props.yAxisType, true)
```

Comment:
>getMinMaxOfYAxis [](http://example.com/codeflow?start=20&length=16)

don't change the function signature of the getMinMax coming from props vs that of utilities